signer/src/utxo.rs

```diff
@@ -387,6 +438,34 @@ impl WithdrawalRequest {
     }
 }
 
+impl TryFrom<model::WithdrawRequest> for WithdrawalRequest {
+    type Error = Error;
+
+    fn try_from(request: model::WithdrawRequest) -> Result<Self, Self::Error> {
+        let amount = request
+            .amount
+            .try_into()
+            .map_err(|_| Error::TypeConversion)?;
+        let max_fee = request
+            .max_fee
+            .try_into()
+            .map_err(|_| Error::TypeConversion)?;
+        let address: Address<NetworkUnchecked> = request
+            .sender_address
+            .parse()
+            .map_err(Error::ParseAddress)?;
+        let address = address.assume_checked(); // We do not validate the request address here
```

Comment:
I don't fully understand this suggestion, but as a blanket statement on all sBTC PRs, the main things we want to prioritize is readability, verifiability, and delivery speed. I say that here because "a little more robust" might not be something to change, and might be more of a "nit".

Given this is the approach we're taking for Deposits with a `try_from` I think the symmetry between our conversion implementation for `Deposit` and `Withdrawal`s from the models using `try_from` is probably the best / most readable approach.

Reply:
Yeah I don't think this is the place to validate the received address, although it doesn't hurt to be conservative here since the risk would be users loosing their funds.

> the main things we want to prioritize is readability, verifiability, and delivery speed

I agree, and I would also add correctness to this list.

Since I'll change the conversion for the `Deposit` I might as well do it here. It's not too much work.

Reply:
Updated ✅